loqusdbapi/utils.py

```diff
@@ -0,0 +1,280 @@
+import logging
+from pathlib import Path
+from typing import List, Union
+
+from cyvcf2 import VCF
+
+from loqusdb.constants import GENOTYPE_MAP
+from loqusdb.models import Variant
+from loqusdb.plugins.mongo.adapter import MongoAdapter
+from loqusdb.utils.profiling import compare_profiles
+from loqusdb.utils.delete import delete
+from loqusdb.build_models.variant import get_variant_id, get_coords, check_par
+from loqusdbapi.exceptions import VCFParserError, ProfileDuplicationError
+from loqusdbapi.models import Case, Individual
+from loqusdbapi.settings import settings
+
+LOG = logging.getLogger("__name__")
+
+
+def get_profiles(adapter: MongoAdapter, vcf_file: Path):
+
```

Comment:
what is this function doing and returning? Type hint and dock string would be nice

Reply:
Same for all functions in this file